src/python_binding/Makefile.am

```diff
@@ -0,0 +1,52 @@
+AM_CPPFLAGS = \
+ -I$(top_srcdir)/node/src \
+ -I$(top_srcdir)/node/src/univalue/include \
+ -I$(top_srcdir)/node/src/consensus \
+ -I$(top_srcdir)/node/src/script \
+ -I$(top_srcdir)/node/src/support \
+ -I$(top_srcdir)/contrib/secp256k1/include \
+ -I$(top_srcdir)/src/common \
+ -I$(top_srcdir)/src/p2p \
+ -I$(top_srcdir)/src/core \
+ -I$(top_srcdir)/contrib/cex\
+ $(PYTHON_CPPFLAGS)
+
+SWIG_SOURCES_CPP = \
+ $(top_srcdir)/node/src/support/cleanse.cpp \
+ $(top_srcdir)/node/src/bech32.cpp \
+ $(top_srcdir)/node/src/primitives/transaction.cpp
```

Comment:
These should be and are the part of libbitcoinconsensus.la. Why it is built again here?